### PR TITLE
Fix 1058 runqing

### PR DESCRIPTION
**Pull request by runqingz**
**IMPORTANT**: By submitting a patch, you agree that your work will be licensed under the [license](https://github.com/umple/umple/blob/master/LICENSE.md) used by the project.

## Description

This is the PR for issue #1058. This PR fix the issue that the error message of error number 19 is not informative enough. This fix will also make the message suggesting potential fix to the ump file to user. 

## Tests

A few tests is also added to make sure the error raise in the correct condition.

### CODE REVIEW

File: cruise.umple/src/en.error [hunk 1]
```diff
@@ -22,7 +22,7 @@
 16: 1, "http://cruise.eecs.uottawa.ca/umple/E016NonImmutab|| (tempMethod != null && tempMethod.isIsAbstract())leSuperclass.html", Immutability rules prevent class '{0}' from subclassing '{1}' ;
 17: 2, "http://cruise.eecs.uottawa.ca/umple/E017NonImmutableBidirectionality.html", Two-way associations may not be immutable ;
 18: 2, "http://cruise.eecs.uottawa.ca/umple/E018ReflexiveImmutable.html", Reflexive immutable associations may not be mandatory ;
-19: 2, "http://cruise.eecs.uottawa.ca/umple/E019DuplicateAssociation.html", Class '{0}' may not have multiple associations with the same name '{1}' ;
+19: 2, "http://cruise.eecs.uottawa.ca/umple/E019DuplicateAssociation.html", There are multiple associations between class '{0}' and class '{1}'. Role names need to be added at '{0}' side to distinguish the different association ends in that class ;
```
Comment (vahdat-ab) [inline cruise.umple/src/en.error]: add a full stop to the end of the sentence.

File: cruise.umple/test/cruise/umple/compiler/024_multiAssocToSameClassWithOneRoleName.ump [hunk 2]
```diff
@@ -0,0 +1,7 @@
+class Person{
+}
+
+class Course{
+  * courseProf -- 1 Person prof;
+  * -- 1..* Person students;
+}
```
Comment (vahdat-ab) [inline cruise.umple/test/cruise/umple/compiler/024_multiAssocToSameClassWithOneRoleName.ump:7]: consider testing more example in the both sides.
For example, 
```
class A{
  0..1  a1-- * B;
  0..1  a2-- * B;
}
class B{}
```
will give the following message in which we have a asociation between `A` and  `A`.
```
Error 19 on line 3 of file "test.ump":
There are multiple associations between class 'A' and class 'A'. Role names need to be added at 'A' side to distinguish the different association ends in that class

```

File: cruise.umple/test/cruise/umple/compiler/024_multiAssocToSameClassWithMultiRoleName.ump [hunk 3]
```diff
@@ -0,0 +1,7 @@
+class Person{
+}
+
+class Course{
+  * courseProf -- 1 Person prof;
+  * courseStudent -- 1..* Person students;
+}
```
Comment (vahdat-ab) [inline cruise.umple/test/cruise/umple/compiler/024_multiAssocToSameClassWithMultiRoleName.ump:7]: Your code doesn't pas the following example:
```
associationClass Friendship {
  * inviterFriend Profile invitee;
  * inviteeFriend Profile inviter;
  status;
}
class Profile {
    name;
}
```
maybe you wanted to have another PR for it?

Comment (runqingz) [reply cruise.umple/test/cruise/umple/compiler/024_multiAssocToSameClassWithMultiRoleName.ump:7]: Yes, that will be another PR

File: cruise.umple/test/cruise/umple/compiler/024_multiAssocToSameClassWithOneRoleName.ump (hunk 2 above)
Comment (vahdat-ab) [reply cruise.umple/test/cruise/umple/compiler/024_multiAssocToSameClassWithOneRoleName.ump:7]: Any improvement on this? The association should be between `A` and `B`.

**Comment by vahdat-ab** [comment]
Thanks!

**Pull request merged**